src/testHelpers/jest-setup.js

```diff
@@ -18,6 +18,9 @@ global.fetch = fetch;
 global.shallow = shallow;
 global.mount = mount;
 
+process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN =
+  'http://localhost.bbc.com:7080';
+process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH = '/static/';
```

Comment:
```suggestion
process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH = '/';
```

Is there any reason why you've set this to `/static/`? The `local.env` value if `/` which seems more relevant when this affects the snapshots.

Reply:
I just copied it from the previous tests but yeah it makes more sense to match the local.env 👍 ... will update now